llvm/lib/Transforms/Coroutines/CoroSplit.cpp

```diff
@@ -1455,6 +1462,62 @@ struct SwitchCoroutineSplitter {
     setCoroInfo(F, Shape, Clones);
   }
 
+  static Function *createNoAllocVariant(Function &F, coro::Shape &Shape,
```

Comment:
We need comment and an example for this function

Reply:
This is done. Thanks.